index.html

```diff
@@ -8,71 +8,56 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <div class="navbar">
+  <nav class="navbar">
     <img class="logo" src="assets/wolox_logo.svg"/>
-    <span class="dev-tools">Developer Tools</span>
-  </div>
+    <h3 class="dev-tools">Developer Tools</h3>
+  </nav>
   <div class="content">
     <div>
       <div class="title">
         <image class="title-image" src="assets/triangle.svg"/>
         <h1>Front-End Tools</h1>
       </div>
       <div class="techs">
-        <div class="tech-container">
-          <div class="background"></div>
-          <div class="tech">
-            <div class="image-container separated">
-              <img src="assets/react_icon.svg" class="image"/>
-            </div>
-            <span class="tech-text separated">ReactJS</span>
-            <span class="release-date tech-text separated">Release: 2013</span>
-            <span class="tech-text separated">ReactJS is an open-source (with caveats) JavaScript library for building user interfaces.</span>
+        <div class="tech">
+          <div class="image-container separated">
+            <img src="assets/react_icon.svg" class="image"/>
           </div>
+          <span class="tech-text separated">ReactJS</span>
+          <span class="release-date tech-text separated">Release: 2013</span>
```

Comment:
The `release-date` class should be named `bold`, since it's the only thing that's being altered :smile_cat: